src/index.js

```diff
@@ -7,19 +7,16 @@ require('@dialonce/boot')({
   LOGS_TOKEN: process.env.LOGS_TOKEN,
   BUGS_TOKEN: process.env.BUGS_TOKEN
 });
-
 require('events').EventEmitter.prototype._maxListeners = process.env.MAX_EMITTERS || 20;
 
-const hostnameFallback = uuid.v4();
-
-/* eslint no-param-reassign: "off" */
 /* eslint global-require: "off" */
 module.exports = (config) => {
+  let configuration = Object.assign({}, config);
   // we want to keep retrocompatibility with older configuration format for a while (until 3.0.0)
   // everything in here is deprecated
-  config = retrocompat(config);
-
-  config = Object.assign({
+  configuration = retrocompat(configuration);
+  const hostnameFallback = uuid.v4();
```

Comment:
since hostnameFallback var is not used no need to declare it, just use uuid.v4();